pkg/test/resourcefixture/testdata/basic/composer/v1alpha1/composerenvironment/_generated_object_composerenvironment.golden.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: composer.cnrm.cloud.google.com/v1alpha1
+kind: ComposerEnvironment
+metadata:
+  finalizers:
+  - cnrm.cloud.google.com/finalizer
+  - cnrm.cloud.google.com/deletion-defender
+  generation: 2
+  labels:
+    cnrm-test: "true"
+  name: composerenvironment-${uniqueId}
+  namespace: ${uniqueId}
+spec:
+  labels:
+    test: "true"
+  location: us-central1
+  projectRef:
+    external: ${projectId}
+status:
+  conditions:
+  - lastTransitionTime: "1970-01-01T00:00:00Z"
+    message: The resource is up to date
+    reason: UpToDate
+    status: "True"
+    type: Ready
+  externalRef: projects/${projectId}/locations/us-central1/environments/composerenvironment-${uniqueId}
+  observedGeneration: 2
+  observedState:
+    config:
+      airflowByoidURI: https://123456qwert-dot-us-central1.composer.byoid.googleusercontent.com
```

Comment:
Suggest changing this API from `Byoid` to `BYOID`.

Reply:
Addressing in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3953.